packages/reactive-dict/reactive-dict.js

```diff
@@ -94,19 +103,19 @@ _.extend(ReactiveDict.prototype, {
     // XXX we could allow arrays as long as we recursively check that there
     // are no objects
     if (typeof value !== 'string' &&
-        typeof value !== 'number' &&
-        typeof value !== 'boolean' &&
-        typeof value !== 'undefined' &&
-        !(value instanceof Date) &&
-        !(ObjectID && value instanceof ObjectID) &&
-        value !== null)
+      typeof value !== 'number' &&
```

Comment:
Understood.  No styleguide scripts.  Hand editing.  I just disabled it from my editor while I do any more edits.